packages/passport/src/index.ts

```diff
@@ -0,0 +1,301 @@
+import type { WalletInit } from '@web3-onboard/common'
+import type { AuthenticatedHeaders } from '@0xpass/passport/dist/types'
+import type { PassportOptions } from 'types'
+import { Network } from '@0xpass/passport'
```

Comment:
```suggestion
import type { Network } from '@0xpass/passport'
```

Reply:
for this `Network` is an enum, so re-exporting it as a type leads to errors in the app demo build because it's being used as a value I believe. So have to re-export it this way, as to not need  `0xpass/passport` as a dependency in the demo app.

for reference can also see capsule implementation with `Environment` https://github.com/blocknative/web3-onboard/blob/76a38237a2d5e71cce73ecbf979cf249a532488d/packages/capsule/src/index.ts

Reply:
Ah got it, thanks for the clarity